parsec/core/cli/import_device.py

```diff
@@ -0,0 +1,70 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2016-2021 Scille SAS
+from pathlib import Path
+
+import click
+
+from parsec.cli_utils import cli_exception_handler, spinner
+from parsec.core import CoreConfig
+from parsec.core.cli.utils import (
+    cli_command_base_options,
+    core_config_options,
+    format_available_devices,
+)
+from parsec.core.local_device import load_device_with_password
+from parsec.core.recovery import create_new_device_from_original
+from parsec.core.types import LocalDevice
+from parsec.utils import trio_run
+
+
+async def _import_device(
+    config: CoreConfig,
+    destination: Path,
+    recovery_password: str,
+    new_password: str,
+    device_label: str,
+) -> None:
+
+    async with spinner("Loading recovery device with password"):
+        recovery_device: LocalDevice = load_device_with_password(
```

Comment:
`load_device_with_password` has return typing in it signature, so no need to specify the typing for `recovery_device` here